app/server/utilityFunctions.js

```diff
@@ -0,0 +1,13 @@
+import jwksClient from "jwks-rsa";
```

Comment:
quotation consisteny


```suggestion
import jwksClient from 'jwks-rsa';
```